.github/workflows/release_npm_version.yml

```diff
@@ -36,6 +36,12 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
 
+      - name: Publish to npm type script
+        run: npm publish --tag latest
```

Comment:
This is a repeat of the previous step. Should that be removed?

Reply:
Because this is in the types directory it should publish just the types project